packages/istio/charts/istio-1.6.3/templates/clusterrole.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ .Release.Name }}-istioctl
```

Comment:
This should match however istio does ClusterRoles, guessing something more like `istio-installer` as long as that doesn't already exist. 
It also needs to be appended by ranchers version of the chart as helm can't reinstall any existing resources it didn't install.